### ISSUE TITLE
Support all input devices with X Input Extension

### ISSUE THREAD

**Issue by locriacyber** [QubesOS/qubes-issues#7324]
### The problem you're addressing (if any)

- https://github.com/QubesOS/qubes-issues/issues/7316
- #4986 
- support for game controllers

### The solution you'd like

Add XInput Devices beside USB Devices to a list of devices to be attached.
One input device can be attached to multiple VMs at once.
By default, a filtered version of mouse and keyboard will be attached to every VM.

### The value to a user, and who that user might be

- use external input device without usb passthrough
- support for multiple input devices, like multiple keyboards
- be able to play first-person games with mouse
- be able to play games using game controller

**Comment by DemiMarie** [comment]
Should this be handled through the GUI protocol?

**Comment by locriacyber** [comment]
Yes. This will likely be a major change to the GUI protocol, where each device has its own id.
